### PR TITLE
Reworks sensory.dm

### PR DESCRIPTION
**Pull request by LanCartwright**
:cl: 
tweak:Removes Synapzitine from sensory restoration.
rscadd:Adds Synaphydramine.
tweak:Adds Synaphydramine to sensory restoration.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -89,6 +89,24 @@
...
+	M.drowsyness = max(M.drowsyness-5, 0)
+	if(holder.has_reagent("mindbreaker"))
+		holder.remove_reagent("mindbreaker", 5)
+	if(holder.has_reagent("histamine"))
+		holder.remove_reagent("histamine", 5)
+	M.hallucination = max(0, M.hallucination - 10)
+	if(prob(30))
+		M.adjustToxLoss(1, 0)
+	..()
+	. = 1
```
Comment (phil235) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Technically, the `. = 1` should only be after `M.adjustToxLoss(1, 0)`

File: code/datums/diseases/advance/symptoms/sensory.dm [hunk 2]
```diff
@@ -80,24 +79,23 @@ Bonus
 		var/mob/living/M = A.affected_mob
 		switch(A.stage)
 			if(1)
-				M << "<span class='warning'>You can't taste a thing.</span>"
```
Comment (phil235) [inline code/datums/diseases/advance/symptoms/sensory.dm]: why remove that message?

Comment (LanCartwright) [inline code/datums/diseases/advance/symptoms/sensory.dm]: It was a useless message, but having a message, or any message in stage 5 will cause it to repeat over, and over, and over again and I felt I should bump any stage 5 messages to stage 4, and so on so forth. Stage 1 had a useless message so I dropped it.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 3]
```diff
@@ -86,8 +86,26 @@
 	M.hallucination = max(0, M.hallucination - 10)
 	if(prob(30))
 		M.adjustToxLoss(1, 0)
+		. = 1
+	..()
+
+/datum/reagent/medicine/synaphydramine
+	name = "Diphen-Synaptizine"
+	id = "synaphydramine"
+	description = "Reducing drowsiness and hallucinations, purges Histamine from body."
```
Comment (KorPhaeron) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: >reducing
>purges

Either reducing and purging or reduces and purges

**Comment by Core0verload** [comment]
Adding a new ultra special chem just to satisfy someone's "balance" pain? Don't like it at all.

**Comment by LanCartwright** [comment]
Don't worry I have plans after this;
Histamine and Mindbreaker overdose redo.
add recipe for synap hydramine
Add a traitor non infectious Chems.
Add new overdose based viruses.

**Pull request merged**